src/tests/daos_racer.c

```diff
@@ -473,7 +473,7 @@ main(int argc, char **argv)
 			oid = racer_oid_gen(idx);
 			rc = daos_obj_verify(ts_ctx.tsc_coh, oid,
 					     DAOS_EPOCH_MAX);
-			if (rc == -DER_NONEXIST) {
+			if (rc == -DER_NONEXIST || rc == -DER_INVAL) {
```

Comment:
hmm, I thought there is better way to valid OID, like query the pool map information and check the available target etc.

Reply:
Seems right. I will update the patch for that.